modules/files/files.sh

```diff
@@ -19,17 +19,17 @@ if [[ ${#FILES[@]} -gt 0 ]]; then
     echo "Adding files to image"
     for pair in "${FILES[@]}"; do
       # Support for legacy recipe format to satisfy transition period to new source/destination recipe format
-      if [[ $(echo $pair | yq '.source') == "null" || -z $(echo $pair | yq '.source') ]] && [[ $(echo $pair | yq '.destination') == "null" || -z $(echo $pair | yq '.destination') ]]; then
+      if [[ $(echo $pair | jq -r 'try .["source"]') == "null" || -z $(echo $pair | jq -r 'try .["source"]') ]] && [[ $(echo $pair | jq -r 'try .["destination"]') == "null" || -z $(echo $pair | jq -r 'try .["destination"]') ]]; then
         echo "ATTENTION: You are using the legacy module recipe format"
         echo "           It is advised to switch to new module recipe format,"
         echo "           which contains 'source' & 'destination' YAML keys"
         echo "           For more details, please visit 'files' module documentation:"
         echo "           https://blue-build.org/reference/modules/files/"
-        FILE="$PWD/$(echo $pair | yq 'to_entries | .[0].key')"
-        DEST=$(echo $pair | yq 'to_entries | .[0].value')
+        FILE="$PWD/$(echo $pair | jq -r 'to_entries | .[0].key')"
+        DEST=$(echo $pair | jq -r 'to_entries | .[0].value')
       else
-        FILE="$PWD/$(echo $pair | yq '.source')"
-        DEST=$(echo $pair | yq '.destination')
+        FILE="$PWD/$(echo $pair | jq -r 'try .["source"]')"
```

Comment:
What is the try syntax and surrounding the key in [ ]?

Reply:
I switched to simpler syntax now.

I used syntax with brackets because I remember when I was making `gnome-extensions` module that 1 key couldn't be extracted without using brackets. So as a fail-safe mechanism, I used this.

However, it's better to keep it simple imo, so I removed bracket syntax.

Reply:
I had the same concern, and would have suggested the same simplification. Glad to be on the same page.